.github/workflows/win_msvc.yml

```diff
@@ -5,8 +5,8 @@ on:
     branches: [ master ]
     tags: ['*']
     paths-ignore: ['**.md']
-  pull_request:
-    branches: [ master ]
+  # pull_request:
```

Comment:
Why is this necessary?

Reply:
It seems that `vcpkg` wants to rebuild a lot due to an update or changes in this PR. Contributors cannot upload packages, so each run will take a long time. I'll enable it when we merge this into master.